cognite_toolkit/_cdf_tk/commands/_profile.py

```diff
@@ -185,6 +187,251 @@ def _as_cell(self, value: PendingCellValue) -> str | Spinner:
+            return partial(aggregator.count, data_set_external_id=data_set_external_id, hierarchy=self.hierarchy)
+        elif col == self.Columns.Transformations:
+            data_set_external_id = rest[-1] if len(rest) > 0 else None
+            if not data_set_external_id:
+                raise ValueError(f"DataSet external ID is required for {row} in column {col}.")
+            return partial(aggregator.used_transformations, data_set_external_ids=[data_set_external_id])
+        elif col == self.Columns.RowCount:
+            db_table = rest[-1] if len(rest) > 0 else None
+            if not isinstance(db_table, str) or "." not in db_table:
+                raise ValueError(f"Database and table name are required for {row} in column {col} in {rest}.")
+            db_name, table_name = db_table.split(".")
+            return partial(
+                client.raw.profile,
+                database=db_name,
+                table=table_name,
+                limit=self.profile_row_limit,
+                timeout_seconds=self.profile_timeout_seconds,
+            )
+        raise ValueError(f"Unexpected API Call for row {row} and column {col}.")
+
+    def format_result(self, result: object, row: str, col: str) -> CellValue:
+        if isinstance(result, int | float | bool | str):
+            return result
+        elif col == self.Columns.DataSets:
+            return result[0] if isinstance(result, list) and result and isinstance(result[0], str) else None
+        elif col == self.Columns.Transformations:
+            if isinstance(result, list) and len(result) > 0 and isinstance(result[0], Transformation):
+                return f"{result[0].name} ({result[0].external_id})"
+            return None
+        elif col == self.Columns.RowCount:
+            if isinstance(result, RawProfileResults):
+                return result.row_count
+            return None
+        raise ValueError(f"unexpected result type {type(result)} for row {row} and column {col}.")
+
+    def update_table(
+        self,
+        current_table: PendingTable,
+        result: object,
+        row: str,
+        col: str,
+    ) -> PendingTable:
+        handlers: Mapping[str, Callable[[PendingTable, object, str], PendingTable]] = {
+            self.Columns.Count: self._update_count,
+            self.Columns.DataSets: self._update_datasets,
+            self.Columns.DataSetCount: self._update_dataset_count,
+            self.Columns.Transformations: self._update_transformations,
+            self.Columns.RowCount: self._update_row_count,
+        }
+        handler = handlers.get(col)
+        if handler:
+            return handler(current_table, result, row)
+        return current_table
+
+    def _update_count(
+        self,
+        current_table: PendingTable,
+        result: object,
+        row: str,
+    ) -> PendingTable:
+        new_table: PendingTable = {}
+        for (r, c), value in current_table.items():
```

Comment:
Nitpick: 
If possible, can you avoid single char variable names? Affects code readability, in general.